cmd/minishift/cmd/start.go

```diff
@@ -422,13 +393,9 @@ func initClusterUpFlags() {
 	clusterUpFlagSet.Bool(metrics, false, "Install metrics (experimental)")
 	clusterUpFlagSet.Bool(logging, false, "Install logging (experimental)")
 	clusterUpFlagSet.String(openshiftVersion, version.GetOpenShiftVersion(), fmt.Sprintf("The OpenShift version to run, eg. %s", version.GetOpenShiftVersion()))
-}
-
-// initProxyFlags create the CLI flags which needs to be passed for proxy
-func initProxyFlags() {
-	proxyFlagSet.String(httpProxy, "", "HTTP proxy for virtual machine (In the format of http://<username>:<password>@<proxy_host>:<proxy_port>)")
-	proxyFlagSet.String(httpsProxy, "", "HTTPS proxy for virtual machine (In the format of https://<username>:<password>@<proxy_host>:<proxy_port>)")
-	proxyFlagSet.String(noProxyList, "", "List of hosts or subnets for which proxy should not be used.")
+	clusterUpFlagSet.String(httpProxy, "", "HTTP proxy used for downloading artefact and configure Docker as well as OpenShift (In the format of http://<username>:<password>@<proxy_host>:<proxy_port>). Overrides a potential HTTP_PROXY setting in the enviroment.")
+	clusterUpFlagSet.String(httpsProxy, "", "HTTPS proxy used for downloading artefact and configure Docker as well as OpenShift (In the format of https://<username>:<password>@<proxy_host>:<proxy_port>). Overrides a potential HTTPS_PROXY setting in the enviroment.")
+	clusterUpFlagSet.String(noProxyList, "", "List of hosts or subnets for which no proxy should be used.")
```

Comment:
They do belong here. Adding them to this flag set allows me to get rid oh the awkward _setOcProxy_ which was just a way to make sure the flags are set in cluster up. We need it in clusterUpFalgSet, since the way we generate the parameters to pass to 'cluster up' is via iterating this set.

This has no bearing to the fact that the use the values somewhere else as well. This was and is handled via _shellProxyEnv_